virtool/postgres.py

```diff
@@ -63,5 +63,10 @@ def to_dict(self):
 
```

Comment:
Move this `postgres.py` module into `virtool.pg.utils` with your other code in this PR.